recipe/meta.yaml

```diff
@@ -79,6 +80,7 @@ outputs:
       commands:
         - test -f $PREFIX/lib/libcurl.a  # [not win]
         - if exist %LIBRARY_LIB%\libcurl_a.lib (exit 0) else (exit 1)  # [win]
+        - test -f %LIBRARY_BIN%\curl.exe  # [win]
```

Comment:
I think you want this in the next one: `curl`, not `libcurl-static.  It'll fail here.